### DISCUSSION TITLE
Make bookmarks help and feedback side bar disappear

### DISCUSSION THREAD

**Discussion by FrankGDot** [alefragnani/vscode-bookmarks#381] [Q&A] [answered]
Hello,

i do not get rid off the the BOOKMARKS:HELP AND FEEDBACK cutout of the sidebar. Instead i want the sidebar show up bookmarks list from all files.

**Comment by alefragnani** [comment, marked as answer]
Hi @FrankGDot,

Are you referring to a _duplicate_ Bookmarks icon appearing at the Side Bar? 

If that's the case, I'm not sure _why_ it happens, because the new Panel is defined/created to the same View/Icon, and VS Code is the one who takes care. It seems a similar issue when you update an extension, and its Icon simply disappears from the Side Bar. Sometimes you have to reinstall the extension, to "fix".

In this case, there is nothing the extension can do to "fix". That panel should be visible at the bottom of the Bookmarks View, so you could simply _move_ it to that location.

Hope this helps
